src/arrays.js

```diff
@@ -1,49 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) 
+  return array[index - array.length];
+  return array[index]
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString()
 };
 
-const csvStringToArray = string => {
-  // your code here
-};
+function csvStringToArray(string) {
+  return string.split(",", 5)
+}
 
-const addToArray = (element, array) => {
-  // your code here
+const addToArray = (elementX, arrayX) => {
+  arrayX.push(elementX)
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
+const addToArray2 = (NEWelement, ORIGarray) => {
+  let addToArray3;
+  addToArray3 = ORIGarray.concat(NEWelement)
+  return addToArray3
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+ return array.splice(index,1)
 };
 
+
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String)
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(function(A){return A.toUpperCase();})
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+   return strings.map(function(A){return A.split("").reverse().join("");})
 };
 
 const onlyEven = numbers => {
-  // your code here
+//const numbers2 = numbers.filter(a => a % 2 === 0); works, replaced filter with map,but this gives T or F//
+return numbers.filter(a => a % 2 === 0);
 };
 
+//splice() method mutates an array and returns the removed items. Splice example with no mutation.
```

Comment:
Dont need to include comments like this, you have a working answer so dont need to add more work to the file with things that dont quite work or were work in progress